.github/workflows/test.yaml

```diff
@@ -50,3 +50,23 @@ jobs:
       - name: Save logs
         if: always()
         uses: ./.github/actions/save-logs
+
+  test-upgrade:
+    if: ${{ inputs.package == 'all' }}
+    runs-on: "uds-ubuntu-big-boy-8-core"
+    timeout-minutes: 30
```

Comment:
Honest question since I'm new to Core and more familiar with DUBBD:  is 30 minutes enough to install and then upgrade?

Reply:
Good question - typically the install on its own takes around 10-11 minutes. Given that the upgrade doesn't have to redo the k3d/zarf init pieces it should be relatively safe to stick with 30 minutes I hope - we haven't really done much testing of upgrades but locally I was seeing things generally take <15 minutes with a few packages cycling their pods.

The timeout is meant to keep us from a big runner eating up minutes if something is hanging - I'd prefer to start small and increase if it seems necessary with real upgrades. Open to other opinions though - not sure if others have tested upgrades and seen different results.

Reply:
All makes sense to me, thanks for the explanation!